packaging/helm-charts/helm3/strimzi-kafka-operator/crds/041-Crd-kafkaconnect.yaml

```diff
@@ -1910,3 +1910,13 @@ spec:
                   type: integer
                   description: The current number of pods being used to provide this resource.
               description: The status of the Kafka Connect cluster.
+            rack:
```

Comment:
This seems wrong. The rack section already is here. This file should not change.

Reply:
Right, too much refactoring.